src/Components/AllSightings.js

```diff
@@ -1,20 +1,89 @@
 import '../Styles/AllSightings.scss';
 import SightingCard from './SightingCard';
-import React, {useState} from 'react';
-import bigfoot from '../assets/bigfoot.jpg';
+import React, { useState, useContext, useEffect } from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { ErrorContext } from '../Context/ErrorContext'
+import { NavigationContext } from '../Context/NavigationContext'
+
+const GET_ALL_SIGHTINGS = gql`
+  query GetAllSightings{
+    getCryptids{
+      name
+      sightings {
+        id
+        location
+        image
+      }
+    }
+  }
+`
 
 const AllSightings = () => {
-  const [ display, setDisplay ] = useState(false)
+  const { data, error, loading } = useQuery(GET_ALL_SIGHTINGS)
```

Comment:
Smart to set pageData for logic to show all sightings or cryptid specific sighting